### PR TITLE
Add cpr

### PR DESCRIPTION
**Pull request by Karasuni**
Added a newer cross-platform `cp` package.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -68,6 +68,7 @@ Utilities to perform common command-line tasks without worrying about cross-plat
 - [rimraf](https://github.com/isaacs/rimraf) - Delete files or directories; like `rm -rf`.
 - [del-cli](https://github.com/sindresorhus/del-cli) - Safer file and folder deletion.
 - [mkdirp](https://github.com/substack/node-mkdirp) - Create a directory, creating parent directories if needed; like `mkdir -p`.
+- [cpr](https://github.com/davglass/cpr) - cp -r for Nodejs
```
Comment (RyanZim) [inline README.md]: Please put cp -r in backticks, like this:

    [cpr](https://github.com/davglass/cpr) - `cp -r` for Nodejs

**Review by RyanZim — CHANGES_REQUESTED** [review]
One formatting nitpick

**Comment by Karasuni** [comment]
You're right, thanks!

**Review by RyanZim — APPROVED** [review]

**Comment by RyanZim** [comment]
Thanks a lot!

**Pull request merged**